istio/0001-Disable-wasm-download-until-we-re-ready-for-it.patch

```diff
@@ -0,0 +1,89 @@
+From 6228d7f09f0486d2ee7cfca0691fb87b18c7183f Mon Sep 17 00:00:00 2001
+From: Brian Avery <bavery@redhat.com>
+Date: Mon, 20 Jul 2020 14:16:54 -0400
+Subject: [PATCH] Disable wasm download until we're ready for it
+
+---
+ bin/init.sh                     | 14 +++++++-------
+ pilot/docker/Dockerfile.proxyv2 |  6 +++---
+ tools/istio-docker.mk           |  8 ++++----
+ 3 files changed, 14 insertions(+), 14 deletions(-)
+
+diff --git a/bin/init.sh b/bin/init.sh
+index 8dfe0e4da..e26dc3f0c 100755
+--- a/bin/init.sh
++++ b/bin/init.sh
+@@ -208,13 +208,13 @@ else
+   ISTIO_ENVOY_NATIVE_PATH=${ISTIO_ENVOY_LINUX_RELEASE_PATH}
+ fi
+ 
+-# Donwload WebAssembly plugin files
+-WASM_RELEASE_DIR=${ISTIO_ENVOY_LINUX_RELEASE_DIR}
+-for plugin in stats metadata_exchange
+-do
+-  FILTER_WASM_URL="${ISTIO_ENVOY_BASE_URL}/${plugin}-${ISTIO_ENVOY_VERSION}.wasm"
+-  download_wasm_if_necessary "${FILTER_WASM_URL}" "${WASM_RELEASE_DIR}"/"${plugin//_/-}"-filter.wasm
+-done
++# Because we separate Istio and proxy builds, we don't need to have these files
```

Comment:
Instead of commenting, you could just delete this block.

Reply:
This is all temporary fixes until we can fix the integration tests. So I'd like to merge as is if it's good.